oscarapi/apps/basket/abstract_models.py

```diff
@@ -0,0 +1,101 @@
+from django.conf import settings
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from oscar.apps.basket.abstract_models import AbstractBasket as _AbstractBasket, AbstractLine as _AbstractLine
+from oscar.apps.basket.managers import OpenBasketManager, SavedBasketManager
+from oscar.core.utils import get_default_currency
+
+from oscarapi.apps.basket.managers import EditableBasketManager
+from oscarapi.apps.basket.utils import get_basket
+
+
+__all__ = ('AbstractBasket', 'AbstractLine')
+
+
+class AbstractBasket(_AbstractBasket):
```

Comment:
I've heard that remark before. Some logic needs to be added to the basket model, I've had it in other places before, but part of this application is an extension on basket. I want that to be clear. Now I agree with you that this is not the most convenient solution. I think I might get away with using a proxy model, so I'll give that a try, stay tuned!